### PR TITLE
CORTX-34091: Disabled parity filtering of bytecount

### PR DESCRIPTION
**Pull request by AbhishekSahaSeagate**
Signed-off-by: Abhishek Saha <abhishek.saha@seagate.com>

### CODE REVIEW

**Comment by AbhishekSahaSeagate** [comment]
https://github.com/Seagate/cortx-motr/pull/2134#issuecomment-1240256607

File: hax/hax/bytecount.py [hunk 1]
```diff
@@ -105,20 +104,18 @@ def _calculate_bc_per_pver(
                 else:
                     pver_bc[pver_fid] = byte_count
 
-        LOG.info('Bytecount with parity buffer: %s', pver_bc)
+        LOG.info('ABK: Bytecount: %s', pver_bc)
```
Comment (pavankrishnat) [inline hax/hax/bytecount.py:107]: kindly remove Personal/debug messages.

File: hax/hax/bytecount.py [hunk 2]
```diff
@@ -105,20 +104,18 @@ def _calculate_bc_per_pver(
...
 
         bc_without_parity: Dict[str, int] = {}
         for pver, bc in pver_bc.items():
             bc_without_parity[pver] = \
                 bc - self._get_parity_buffers(bc, pver_state[pver])
-        LOG.info('Bytecount without parity buffer: %s', bc_without_parity)
+        LOG.info('ABK: Bytecount without parity buffer: %s', bc_without_parity)
```
Comment (pavankrishnat) [inline hax/hax/bytecount.py:113]: Here also, remove Personal/debug messages.

**Comment by d-nayak** [comment]
Closing this since this was part of a PoC for the memory leak issue in motr fs stats and byte count. The associated ticket (CORTX-34091) is already closed out.

**Pull request closed without merging**